tests/Mockery/CallableSpyTest.php

```diff
@@ -29,7 +29,8 @@ class CallableSpyTest extends MockeryTestCase
     /** @test */
     public function it_verifies_the_closure_was_called()
     {
-        $spy = spy(function() {});
```

Comment:
Looks like it still does this weird splitting of braces?

Reply:
I've removed outdated commits.👍🏽